Sources/NIO/System.swift

```diff
@@ -535,6 +556,50 @@ internal extension Posix {
     }
 }
 
+internal extension Posix {
+    static func cmsgFirstHeader(inside msghdr: UnsafePointer<msghdr>?) -> UnsafeMutablePointer<cmsghdr>? {
```

Comment:
This argument is forbidden to be `nil` (assert in the C layer), so let's not accept `nil` here.